app/src/test/java/com/codesoom/assignment/TaskControllerTest.java

```diff
@@ -0,0 +1,187 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.application.TaskService;
+import com.codesoom.assignment.domain.Task;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.patch;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SuppressWarnings({"InnerClassMayBeStatic", "NonAsciiCharacters"})
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.MOCK)
+@AutoConfigureMockMvc
+@DisplayName("TaskController 클래스")
+class TaskControllerTest {
+    @Autowired
+    MockMvc mockMvc;
+    @Autowired
+    ObjectMapper objectMapper;
+    @Autowired
+    TaskService taskService;
+
+    Task createTask(String title) throws Exception {
+        Task task = new Task(0, title);
+        MvcResult mvcResult = mockMvc.perform(post("/tasks")
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON)
+                .content(objectMapper.writeValueAsString(task)))
+                .andReturn();
+        return objectMapper.readValue(mvcResult.getRequest().getContentAsString(), Task.class);
+    }
+
+    @AfterEach
+    void clearData() {
+        taskService.clearTasks();
+    }
+
+    @Nested
+    @DisplayName("GET 메소드는")
```

Comment:
http 메소드와 java 메소드가 혼동될 수 있을 것 같습니다. 다음과 같이 수정하면 어떨까요?
```suggestion
    @DisplayName("GET 메소드 리퀘스트는")
```

Reply:
동의합니다! 그 표현이 더 이해하기에 좋을 것 같습니다.